docs/plugins/asg_rollout.md

```diff
@@ -0,0 +1,85 @@
+# Auto-Scaling Group LaunchConfig Rollout Plugin
+
+## Overview
+
+This plugin adds support for rolling out changes to Auto Scaling
+Groups that have happened after a stack update. It supports various
+pre- and post-actions on the instances (using Moonshot's native SSH
+support).
+
+## Example
+
+The ASGRollout class is intended to be used within a method in your
```

Comment:
Yeah, mechanisms are used by the controller. Man, I regret calling them mechanisms.